city-infrastructure-platform/settings.py

```diff
@@ -91,7 +91,7 @@
 LOGIN_REDIRECT_URL = "/admin/"
 LOGOUT_REDIRECT_URL = "/admin/login/"
 SOCIAL_AUTH_TUNNISTAMO_AUTH_EXTRA_ARGUMENTS = {"ui_locales": "fi"}
-WAGTAIL_SITE_NAME = _("City Infrastructure Platform Administration")
+WAGTAIL_SITE_NAME = _("City Infrastructure Platform")
```

Comment:
I might miss something, but are we using WAGTAIL somewhere?

Reply:
No, we are not using Wagtail. But according to source code, that is the only way to localize Admin UI front-page title without completely rewriting it on a view/template basis.